custom/src/main/java/org/kiwix/kiwixmobile/custom/download/CustomDownloadActivity.kt

```diff
@@ -44,6 +45,9 @@ import org.kiwix.kiwixmobile.custom.download.State.DownloadRequired
 import javax.inject.Inject
 
 class CustomDownloadActivity : BaseActivity() {
+  override fun injection(coreComponent: CoreComponent) {
+    coreComponent.inject(this)
```

Comment:
this needs to inject via `customActivityComponent` because it uses Activity in its dependencies